### PR TITLE
font and scss

### PR DESCRIPTION
**Pull request by maialeandra**
I got Sep's handwriting font to work, and then started trying to center the tables and make buttons. The centering is kind of wonky, in that whatever doesn't have the "mixin" is appearing above or on top of the table. It's obviously not perfect but I wanted you to see what I had so we could work the kinks out together. 

Ideally, my vision is that when an admin is logged in, their table will have a thin border in the color of their school logo. So let's say Castle would be looking at Wild Rose with a mint green border, or the admin at Violeta would see a purple border. This visual could come in handy for super users like, Katelyn, who might need to shuffle between schools and don't want to get confused as to which school's files she's looking at. 

In order to implement that feature, I assume we need to check the session id or something to verify which admin is logged in and display the proper school permissions and colors. However, I'm not quite sure how we're going to configure the admin functionality aside from the super admin. 

Once we figure that out, I think it'd be nice to include (as you had mentioned during the design phase) a "Welcome, you are logged in as ____" on the upper right corner next to the log out button.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
     <script src="https://use.fontawesome.com/a35cd94d29.js"></script>
-  </head>
 
-  <body>
     <% if current_admin %>
       <%= link_to 'Log Out', destroy_admin_session_path, method: :delete %>
     <% end %>
+
+    <h1>Wildflower Donations Portal</h1>
```
Comment (ssalzberg) [inline app/views/layouts/application.html.erb]: There shouldn't be any tags like <a> or <h1> in the head section of your page. That stuff should all go in the body.

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -1,15 +1,42 @@
+@mixin centerer {
+  position: absolute;
```
Comment (ssalzberg) [inline app/assets/stylesheets/scaffolds.scss]: you probably don't want to center things with position: absolute all the time. If you're only looking for left/right centering, you can set your table to a fixed width, then use margin: auto

http://www.w3schools.com/css/css_align.asp

File: db/schema.rb [hunk 3]
```diff
@@ -42,6 +42,7 @@
     t.datetime "created_at",     null: false
     t.datetime "updated_at",     null: false
     t.integer  "school_id"
+    t.string   "school_name"
```
Comment (ssalzberg) [inline db/schema.rb]: Why did you add a school_name to Admins?

File: db/schema.rb [hunk 4]
```diff
@@ -50,4 +51,15 @@
     t.datetime "updated_at",  null: false
   end
 
+  create_table "users", force: :cascade do |t|
```
Comment (ssalzberg) [inline db/schema.rb]: Why the Users resource? If anything this probably doesn't belong in this branch since this is about front end css, right?

File: db/schema.rb (hunk 3 above)
Comment (maialeandra) [inline db/schema.rb]: I tried removing the migration like we had done last time and I think I messed it up and re added it again... I have screenshots that show it got removed though so I'm confused.

File: db/schema.rb (hunk 4 above)
Comment (maialeandra) [inline db/schema.rb]: I have a whole other branch dedicated to user flow so I'm not sure how that got in there.

**Comment by ssalzberg** [comment]
👍
